### DISCUSSION TITLE
What is deta.send_email?

### DISCUSSION THREAD

**Discussion by s4m-mo** [deta/cloud-docs#453] [Help] [answered]
I noticed, using VS Code intellisense, that `deta-python` has a function called `send_email`. Does this allow your micro to send emails, or is this for email confirmation, or something else? What email address does it give you?

**Comment by jnsougata** [comment]
it's used internally ig... or for the custom domains.

**Comment by abdelhai** [comment, marked as answer]
this is an old deprecated feature. it won't work for your micros.
